ui/src/app/components/summary/summary.component.html

```diff
@@ -140,7 +140,7 @@
       </svg>
     </div>
     <div class="no-object-message">
-      Click on a converted object name on the Spanner tree panel to view its conversion issues.
+      Click on a converted object name on the <a (click)="spannerTab()">Spanner tree panel</a> to view its conversion issues.
```

Comment:
Hyperlink doesn't appear in the UI: https://drive.google.com/file/d/1uuy_sW0p-m2189gCe-Au4Kb82cUA8krj/view?usp=share_link&resourcekey=0-aDWhcHFoqi4P-59xow3Q7w

Reply:
And this link should be added in the header where conversion assessment appears

Reply:
Updated